qemu/tests/numa_consistency.py

```diff
@@ -51,6 +52,8 @@ def get_vcpu_used_node(numa_node_info, vcpu_thread):
     params['backend_mem'] = "memory-backend-ram"
     params['use_mem'] = "no"
     params['size_mem'] = "1024M"
+    if arch.ARCH in ('ppc64', 'ppc64le'):
+        params['size_mem'] = "4096M"
```

Comment:
Could you explain the reason in the commit message?

Reply:
The original one is too small for Power, so preferring to have a practical and reasonable size, thanks.